### DISCUSSION TITLE
How to enable full screen with zoom to full screen?

### DISCUSSION THREAD

**Discussion by DarkyWorker** [otya128/winevdm#1292] [Q&A]
Hello! Now all games are running in a small window in the middle of a black screen. Is it possible to scale the game to full screen?
![Desktop Screenshot 2023 04 30 - 12 18 38 61](https://user-images.githubusercontent.com/17611383/235340922-abaafbc9-ea05-41f0-ac60-94c58a25989e.png)

**Comment by cracyc** [comment]
You can lower your screen resolution or try using a magnifier, see https://github.com/otya128/winevdm/issues/561 .
